ui/src/components/VolumeOverviewTab.js

```diff
@@ -360,31 +360,32 @@ const VolumeDetailCard = (props) => {
               />
             </AlertsCounterContainer>
           )}
-          {condition === VOLUME_CONDITION_LINK && (
-            <VolumeUsage>
-              <VolumeSectionTitle>
-                {intl.formatMessage({ id: 'usage' })}
-              </VolumeSectionTitle>
-              {volumeUsagePercentage !==
-              intl.formatMessage({ id: 'unknown' }) ? (
-                <ProgressBarContainer>
-                  <ProgressBar
-                    size="large"
-                    percentage={volumeUsagePercentage}
-                    topRightLabel={`${volumeUsagePercentage}%`}
-                    bottomLeftLabel={`${volumeUsageBytes} USED`}
-                    bottomRightLabel={`${formatSizeForDisplay(
-                      storageCapacity,
-                    )} TOTAL`}
-                    color={theme.infoSecondary}
-                    backgroundColor={theme.buttonSecondary}
-                  />
-                </ProgressBarContainer>
-              ) : (
-                <LoaderContainer size="small" />
-              )}
-            </VolumeUsage>
-          )}
+          {volumeUsagePercentage !== undefined &&
+            condition === VOLUME_CONDITION_LINK && (
+              <VolumeUsage>
+                <VolumeSectionTitle>
+                  {intl.formatMessage({ id: 'usage' })}
+                </VolumeSectionTitle>
+                {volumeUsagePercentage !==
+                intl.formatMessage({ id: 'unknown' }) ? (
```

Comment:
Not really related to this specific bug fix, but I think we can remove this checking.

Reply:
I trust you on this one, i'm not 100% sure that `volumeUsagePercentage`  as never this value

Reply:
I think it should be OK.. We got the `volumeUsagePercentage` from the `usage` which we compute here:

NodeVolumesUtil.js
```
        usage:
          volumeUsedCurrent && volumeCapacityCurrent
            ? (
                (volumeUsedCurrent?.value[1] / volumeCapacityCurrent.value[1]) *
                100
              ).toFixed(2)
            : undefined,
 ```